src/app/modules/tabs/tab/tab.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="ds-c-tabs__item {{className}}" role="tab"
```

Comment:
Just curious; why _tab_ component? Seems like a logical loop in the _**tabs**_ component where all tabs would share the same callback.

Reply:
We were following the naming conventions and code organization set up by the cms design system - https://design.cms.gov/components/tabs/